libr/bin/p/bin_smd.c

```diff
@@ -142,14 +142,23 @@ static RList *symbols(RBinFile *bf) {
 	RList *ret = NULL;
 	const char *name;
 	SMD_Header *hdr;
-	int i;
+	int left, i;
 
 	if (!(ret = r_list_new ())) {
 		return NULL;
 	}
 	ret->free = free;
 	// TODO: store all this stuff in SDB
-	hdr = (SMD_Header *) (bf->buf->buf + 0x100);
+#if 0
```

Comment:
please let's just remove this `#if 0` if the code does not work well. We have git to retrieve it, no need to keep it here.

Reply:
i left it to decide if its better to parse a header filled by zeros or cancel if not enough bytes